### ISSUE TITLE
Proper keyboard focus boundary is not visible on the ‘Copy to clipboard’ button.

### ISSUE THREAD

**Issue by msftedad** [jperasmus/docsify-copy-code#71]
**Test Environment:**
OS: Windows11
OS Version: 23H2 (Build 25905.1000)
Browser: Edge Dev (Version 117.0.2007.0 (Official build) dev (64-bit))
URL: [PWABuilder - Packaging for the Google Play Store](https://docs.pwabuilder.com/#/builder/android)
Screen reader: Narrator

**Steps to reproduce**

1. Open the URL: [PWABuilder - Packaging for the Google Play Store](https://docs.pwabuilder.com/#/builder/android) in browser.
2. Navigate to ‘Copy to clipboard’ button through tab key.
3. Observe the issue.

**Current behaviour**
Keyboard focus boundary is not visible on the ‘Copy to clipboard’ button present at the page.

**Expected behaviour**
Proper keyboard focus boundary should be visible on the ‘Copy to clipboard’ button.

**Attachments:**
<img width="959" alt="Proper keyboard focus boundary is not visible on the ‘Copy to clipboard’ button1 (1)" src="https://github.com/jperasmus/docsify-copy-code/assets/93735775/95071a77-ae90-4e2f-87cc-207ac10388a9">
<img width="960" alt="Proper keyboard focus boundary is not visible on the ‘Copy to clipboard’ button (1)" src="https://github.com/jperasmus/docsify-copy-code/assets/93735775/0780b31c-e5f6-4d5c-8275-69eb0cc947b9">

**Comment by msftedad** [comment]
Verified this issue is still reproduce on latest environment

Pre-requisite:
Open System Settings.
Select 'Accessibility' tab.
Select 'Color Contrast theme'.
In the 'Color Contrast theme' combo box select 'Aquatic/Desert' theme.
Test Environment:
OS Build: Windows 11 version 23H2 (26040.1000)
Edge Browser: Version 122.0.2365.3(Official build) dev (64-bit)
URL: [PWA Starter - Routing and Navigation (pwabuilder.com)](https://docs.pwabuilder.com/#/starter/adding-content?id=adding-pages)
![copy to clipboard](https://github.com/jperasmus/docsify-copy-code/assets/93735775/6b182770-d2de-49d8-b532-bd7becb213bc)

**Comment by msftedad** [comment]
@jperasmus, Can you please guide on which we need to add PR?
![image](https://github.com/jperasmus/docsify-copy-code/assets/93735775/371ef36c-8440-4933-84ca-1f21d4d4e44b)

**Comment by msftedad** [comment]
@jperasmus, we are unable to create PR for this issue. Could you please help us on creating the PR?

**Comment by jperasmus** [comment]
@msftedad Are you running into permission issues or what is the problem you're experiencing? If it is permission issues, you are likely trying to push directly to a branch on this repo. That won't work. What you should do instead is fork this repo and then create the PR from your branch on the fork to the `master` branch of this repo.
